meta-facebook/yv35-rf/src/platform/plat_version.h

```diff
@@ -42,7 +42,7 @@
 
 #define BIC_FW_YEAR_MSB 0x20
 #define BIC_FW_YEAR_LSB 0x23
-#define BIC_FW_WEEK 0x03
+#define BIC_FW_WEEK 0x08
```

Comment:
Line 36 needs to be updated as well to indicate the revision field correctly.  It should be changed from 0x02 to 0x03.  Thanks!

Reply:
I have modified the revision field in the #97d5110 commit.
Many thanks.